.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "trailingComma": "all",
-  "printWidth": 100
+  "printWidth": 100,
+  "proseWrap": "never"
```

Comment:
This will affect all Markdown formatted by Prettier. I don't have a strong feeling one way or the other, just noting it has a more widespread impact than just i18n strings.

Reply:
I'd like to keep these `.yml` as un-wrapped as possible, so I could back this change out but add it to `normalize-yaml` script as an override?

Reply:
I think it's fine if we acknowledge this as a project-wide default, which seems reasonable enough?

But yeah, if needed, I think this could also go back to being able to configure `normalize-yaml` with a specific Prettier configuration if we wanted to behave differently specific for `normalize-yaml` / specific for specific sets of files normalized by `normalize-yaml`.